packages/teams-js/src/public/pages.ts

```diff
@@ -66,12 +74,16 @@ export function tabsNavigateToTabHelper(apiVersionTag: string, tabInstance: TabI
   });
 }
 
-export function returnFocusHelper(apiVersionTag: string, navigateForward?: boolean): void {
+export function returnFocusHelper(
+  apiVersionTag: string,
+  navigateForward?: boolean,
```

Comment:
If a developer specifies a boolean for `navigateForward` AND  a `returnFocusActionItem` in the same function call, what happens?